src/blockchain/network.ts

```diff
@@ -108,20 +108,20 @@ export type GasPrice$ = Observable<BigNumber>;
 export const gasPrice$: GasPrice$ = web3Ready$.pipe(
   switchMap(() => concat(
     bindNodeCallback(web3.eth.getGasPrice)(),
-    onEveryBlock$.pipe(
-      switchMap(() => ajax({
-        url: 'https://ethgasstation.info/json/ethgasAPI.json',
-        method: 'GET',
-        headers: {
-          Accept: 'application/json',
-        },
-        crossDomain: true,
-      })),
-      retryWhen(errors => errors.pipe(delayWhen(() => onEveryBlock$.pipe(skip(1))))),
-      map(({ response }) =>
-        new BigNumber(response.average).times(1.1).times(new BigNumber(10).pow(8))
-      ),
-    )
+    // onEveryBlock$.pipe(
```

Comment:
Can we just remove it ?

Reply:
This is temporary on purpose. After testing we either remove the comment or revert